src/core_modules/capture-core/components/CardList/CardList.types.js

```diff
@@ -0,0 +1,36 @@
+// @flow
```

Comment:
This whole file used to be in the search page. I moved it here since it feels like is a better place to have it. Open to suggestions tho